src/picasso-components/heat-map-highlight/index.js

```diff
@@ -0,0 +1,57 @@
+import KEYS from '../../constants/keys';
+import getImageData from './extract-image-data';
+
+export default {
+  require: ['renderer', 'chart'],
+  renderer: 'canvas',
+
+  render() {
+    const { element } = this.chart;
+    const heatMapCanvas = element.querySelector(`[data-key=${KEYS.COMPONENT.HEAT_MAP}]`);
+    const heatMapHighlightCanvas = element.querySelector(`[data-key=${KEYS.COMPONENT.HEAT_MAP_HIGHLIGHT}]`);
```

Comment:
you can do `this.renderer.element()` to get hold of the element for this component.